storybookjs/storybook
# Angular: Fix angular2-template-loader / raw-loader version conflicts

Issue: https://github.com/storybookjs/storybook/issues/7877

## What I did

As suggested in https://github.com/storybookjs/storybook/issues/7877#issuecomment-536556755 I fixed this issue by modifying the code of `angular2-template-loader`

Though, instead of using `patch-package` I just copied the loader function from the `angular2-template-loader` repository. This package hasn't received any updates in the past 3 years and I contacted the owner to talk about the future of this package.

Also, copying the loader also means we don't have to install another dependency that also needs to be executed as a postinstall script.

## How to test

- Checkout this branch
- Execute the `prepare` script in `app/angular`
- Start the angular example in `examples/angular-cli`
- Go to the `AppComponent` story 
  - If it works you see the initial angular welcome page
  - It does not work if you see an error like this
```
@storybook/angular: Expected 'styles' to be an array of strings
```

## What else

We should make sure this works in Angular 7 and 8 before merging it
